src/js/media-seek-backward-button.js

```diff
@@ -39,7 +39,11 @@ class MediaSeekBackwardButton extends MediaChromeButton {
 
   constructor(options = {}) {
     super({ slotTemplate, ...options });
+  }
+
+  connectedCallback(){
```

Comment:
nit (non-b): a space after is generally used in the code base

```suggestion
  connectedCallback() {
```